babylon/chain.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "babylon",
+  "status": "live",
+  "website": "https://www.babylonchain.io",
+  "network_type": "testnet",
```

Comment:
are we adding the mainnet or testnet? testnets should in the /testnets/ directory and the chainname would be 'babylontestnet'.

Reply:
Thanks for the prompt response. Moved the entire folder to `testnets/babylontestnet` and renamed corresponding places.